### PR TITLE
Add new certs to v4

### PR DESCRIPTION
**Pull request by stechiu**
### Summary of changes

 - This PR contains the new certs

### Checklist

 - [x] Added a changelog entry
 - [ ] Relevant test coverage

### CODE REVIEW

**Comment by jaxdesmarais** [comment]
Removing the Jira ticket from the description

**Comment by stechiu** [comment]
> Removing the Jira ticket from the description

Good call, thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+* Updated expiring pinned vendor SSL certificates
```
Comment (jaxdesmarais) [inline CHANGELOG.md:4]: I think the Android CHANGELOG generally adds a space here:
```suggestion
## unreleased

* Updated expiring pinned vendor SSL certificates
```

**Review by jaxdesmarais — COMMENTED** [review]
We should dig into the lint and unit test failures here

**Review by saperi22 — APPROVED** [review]

**Review by KunJeongPark — APPROVED** [review]

**Pull request merged**